housinghelper/listings/templates/listings/register.html

```diff
@@ -1,137 +1,137 @@
-                            {{form.username}}
-                        </div>
-
-                        <div class="input-group mb-2">
-                            <div class="input-group-append">
-                                <span class="input-group-text"><i class="fas fa-key"></i></span>
-                            </div>
-                            {{form.password1}}
-                        </div>
-
-                        <div class="input-group mb-2">
-                            <div class="input-group-append">
-                                <span class="input-group-text"><i class="fas fa-key"></i></span>
-                            </div>
+						<div class="input-group mb-3">
+							<div class="input-group-append">
+								<span class="input-group-text"><i class="fas fa-user"></i></span>
+							</div>
+							{{form.username}}
+						</div>
+
+						
+
+						<div class="input-group mb-2">
+							<div class="input-group-append">
+								<span class="input-group-text"><i class="fas fa-key"></i></span>
+							</div>
+							{{form.password1}}
+						</div>
+
+						<div class="input-group mb-2">
+							<div class="input-group-append">
+								<span class="input-group-text"><i class="fas fa-key"></i></span>
+							</div>
                             {{form.password2}}
-                        </div>
-
-                        <div class="d-flex justify-content-center mt-3 login_container">
-                            <input class="btn login_btn" type="submit" value="Register Account">
-                        </div>
-                    </form>
-                </div>
-
-                <div class="mt-4">
-                    <div class="d-flex justify-content-center links">
-                        Already have an account? <a href="{% url 'login' %}" class="ml-2">Login</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
-    <script>
-        /* Because i didnt set placeholder values in forms.py they will be set here using vanilla Javascript
-//We start indexing at one because CSRF_token is considered and input field
-*/
-
-        //Query All input fields
-        var form_fields = document.getElementsByTagName('input')
-        form_fields[1].placeholder = 'Username..';
-        form_fields[2].placeholder = 'Enter password...';
-        form_fields[3].placeholder = 'Re-enter Password...';
-
-
-        for (var field in form_fields) {
-            form_fields[field].className += ' form-control'
-        }
-    </script>
+						</div>
+
+				   		<div class="d-flex justify-content-center mt-3 login_container">
+				 			<input class="btn login_btn" type="submit" value="Register Account">
+				   		</div>
+					</form>
+				</div>
+
+                {{form.errors}}
+		
+				<div class="mt-4">
+					<div class="d-flex justify-content-center links">
+						Already have an account? <a href="{% url 'login' %}" class="ml-2">Login</a>
+					</div>
```

Comment:
It would be nice if we had a redirect back to the home page

Reply:
I agree with Muzahid here.

Reply:
Will be working on it.